app/auto-eligibility/routes/users.js

```diff
@@ -0,0 +1,49 @@
+const Joi = require('joi')
+const Boom = require('@hapi/boom')
+const eligibilityDbTable = require('../db/eligibility.db.table')
+
+module.exports = {
+  method: 'GET',
+  path: '/api/users',
```

Comment:
Might be good to look into versioning of the original `api/eligibility` endpoint? 

`api/users` to me suggests it would return all users who have access (maybe for backoffice) and to me is separate to the eligibility check API

Reply:
renamed to /api/businesses